src/nrnmpi/nrnmpidec.h

```diff
@@ -108,7 +107,7 @@ extern void nrnmpi_barrier();
 extern double nrnmpi_dbl_allreduce(double x, int type);
 
 extern void nrnmpi_dbl_allreduce_vec(double* src, double* dest, int cnt, int type);
-extern void nrnmpi_longdbl_allreduce_vec(longdbl* src, longdbl* dest, int cnt, int type);
+extern void nrnmpi_longdbl_allreduce_vec(long double* src, long double* dest, int cnt, int type);
```

Comment:
It's use was only in the hope that an addition reduction would be more independent of order of element addition.

Reply:
thanks!

Reply:
To muddy the waters in respect to whether long double is needed is that I believe Olli replaced it with Kahan summation in several places. I.e.
```
~/neuron/nrn/src/nrniv$ grep -r -i kahan
nvector_nrnthread_ld.cpp:// for use alongside retval in Kahan summation that replaces old long double code
nvector_nrnserial_ld.cpp:    // Use Kahan summation instead of a long double accumulator for better portability.
...
```